src/lib/POWERMGNT/POWERMGNT.h

```diff
@@ -31,13 +31,13 @@
 #endif
 
 #ifdef TARGET_100mW_MODULE
-#define MaxPower 2
-#define DefaultPowerEnum 2
+#define MaxPower PWR_100mW
```

Comment:
That should be `PWR_50mW`.

Reply:
Why the maximum power should be 50mW if the target is 100mW? I know that "2" value corresponds to 50mW in enum but since the target is 100mW I think `PWR_100mW` is the right value. 
Am I missing something?

Reply:
The bare sx1276 can only do 50mW at the high duty cycle ExLRS uses.

Reply:
Ok I'll fix!